### PR TITLE
ungoogled-chromium: 89.0.4389.114 -> 90.0.4430.85

### PR DESCRIPTION
**Pull request by primeos**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/chromium/common.nix [hunk 1]
```diff
@@ -161,15 +161,7 @@ let
     patches = [
       ./patches/no-build-timestamps.patch # Optional patch to use SOURCE_DATE_EPOCH in compute_build_timestamp.py (should be upstreamed)
       ./patches/widevine-79.patch # For bundling Widevine (DRM), might be replaceable via bundle_widevine_cdm=true in gnFlags
-      # ++ optional (versionRange "68" "72") (githubPatch "<patch>" "0000000000000000000000000000000000000000000000000000000000000000")
```
Comment (lukegb) [inline pkgs/applications/networking/browsers/chromium/common.nix:164]: Worth keeping this comment to describe how to use githubPatch?

Comment (primeos) [reply pkgs/applications/networking/browsers/chromium/common.nix:164]: I thought about it but it's not that useful IOM . I might re-add it in the future with a more useful formatting/structure.

Comment (lukegb) [reply pkgs/applications/networking/browsers/chromium/common.nix:164]: SG 👍

**Comment by primeos** [comment]
`nix-build nixos/tests/chromium.nix -A ungoogled` succeeded.

**Pull request merged**